Twig/Recommendation_Twig_Extension.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Twig;
+
+use eZ\Publish\API\Repository\Repository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Templating\DelegatingEngine;
+
+class Recommendation_Twig_Extension extends \Twig_Extension
```

Comment:
@clash82 Check out PSR-2, basically there are two choices to naming:
- either the old PEAR way like used by Twig still (will probably be fixed in Twig 2.0) where `_` translates to path separator
- the other is namespace based where it directly maps to paths with no need for dealing with that in class name anymore